src/styles.css

```diff
@@ -418,4 +413,13 @@ a.dropdown-item:hover {
   .desc-text{
     font-size: 0.95rem;
   }
+
+  label {
```

Comment:
If bulma makes it required that we always need class label with label tag; and has some CSS applied better to have it on .label, over-ridden